Source/Urho3D/Audio/Audio.cpp

```diff
@@ -52,6 +53,43 @@ static const StringHash SOUND_MASTER_HASH("Master");
 
 static void SDLAudioCallback(void* userdata, Uint8* stream, int len);
 
+static int AUDIO_NUM_CHANNELS[] = {
+    6, // Auto, just aim for 5.1
+    1, // mono
+    2, // stereo
+    4, // quadrophonic
+    6, // 5.1
+};
+
+// SM_AUTO is BAD!
+static SpeakerMode CHANNELS_TO_MODE[] = {
+    SPK_AUTO, // invalid actually,
```

Comment:
Why does comment say this is invalid?

Reply:
Ahhh, it's because it's not an actually legitimate mode. It's a stand-in value used for the indexing to walk through the available modes. Perhaps it's less invalid then I had thought as it could be useful to retain separately for the audio device loss/added case.